backtrader/plot/plot.py

```diff
@@ -127,6 +127,9 @@ def plot(self, strategy, figid=0, numfigs=1, iplot=True,
                 matplotlib.use('nbagg')
 
         # this import must not happen before matplotlib.use
+        from IPython import get_ipython
```

Comment:
IPython is not the prerequisite for the backtrader ( as matplotlib - see https://www.backtrader.com/docu/installation/).

So importing the `get_ipython` method may fail - please put this code inside the try..except.